### PR TITLE
[feature] google map in details page, scroll behaviour implemented, g…

### PR DESCRIPTION
**Pull request by sanket-vekariya**
I have implemented google map in details page which shows the relevant country at the bottom,
also added gif as a custom loader, made consistent appBar in the home screen, scroll behavior.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ dependencies:
     sdk: flutter
 
   # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
+  # Use with the CupertminoIcons class for iOS style icons.
```
Comment (ashishkhuraishy) [inline pubspec.yaml:24]: typo?

**Review by ashishkhuraishy — APPROVED** [review]
great work man 👍

File: pubspec.yaml (hunk 1 above)
Comment (sanket-vekariya) [reply pubspec.yaml:24]: oops..😅

Comment (sanket-vekariya) [reply pubspec.yaml:24]: Can I be the contributor if possible ? I can help out with news feed for this project.

**Comment by ashishkhuraishy** [comment]
Hey there, the ios build by githib actions is showing some build error. If you have a mac then can you check and add an ios build on to the repo😄

**Comment by sanket-vekariya** [comment]
Unfortunately I don't have mac system.

File: pubspec.yaml (hunk 1 above)
Comment (ashishkhuraishy) [reply pubspec.yaml:24]: hey sorry i haven't seen this message then. Yah sure you can become a contributor, feel free to add the news feature :)

**Pull request merged**